Templates/SendGrid-Timer-CSharp/function.json

```diff
@@ -0,0 +1,18 @@
+{
+    "bindings": [
+        {
+            "type": "timerTrigger",
+            "name": "myTimer",
+            "schedule": "0 0 23 * * *",
+            "direction": "in"
+        },
+        {
+            "type": "sendGrid",
+            "name": "$return",
+            "direction": "out",
+            "apiKey":  "SendGridApiKey",
+            "from": "Azure Functions <samples@functions.com>",
+            "to": "{CustomerEmail}"
```

Comment:
This won't be populated with the timer trigger, so this sample won't work out of the box.